VAMobile/src/api/vaccines/getVaccineLocation.tsx

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from '@tanstack/react-query'
+
+import { VaccineLocationPayload } from 'api/types'
+import { get } from 'store/api'
+
+import { vaccineKeys } from './queryKeys'
+
+/**
+ * Fetch user VaccinesLocation
+ */
+const getVaccinesLocation = async (locationId: string): Promise<VaccineLocationPayload | undefined> => {
+  const response = await get<VaccineLocationPayload>(`/v0/health/locations/${locationId}`)
+  return response
```

Comment:
It may be cleaner, but I do like having access to the full response in the event something gets added or needs to be changed and more closely models the json response. So I feel like this is fine to sacrifice a little bit of how clean it looks for the improved functionality in the event we need it